### PR TITLE
Multi lang

### PR DESCRIPTION
**Pull request by ilackarms**
- fix `wasme list`
- fix deploy gloo

### CODE REVIEW

**Comment by ilackarms** [comment]
/kick

needed to make `yuvaltest.solo.io/ilackarms/gloo-test:1.3.3-0` public

File: pkg/defaults/defaults.go [hunk 1]
```diff
@@ -13,3 +16,9 @@ func NewDefaultCache() cache.Cache {
 
 	return cache.NewCache(puller)
 }
+
+var (
+	WasmeConfigDir       = os.Getenv("HOME") + "/.wasme"
```
Comment (yuval-k) [inline pkg/defaults/defaults.go]: UserHomeDir

**Comment by ilackarms** [comment]
/kick

404 @yuval-k

**Pull request merged**